presidio-anonymizer/README.MD

```diff
@@ -1 +1,121 @@
-# TODO add the document Omri wrote.
\ No newline at end of file
+# Presidio anonymizer
+
+## Description
+
+The Presidio anonymizer is a Python based service for anonymizing detected PII text
```

Comment:
not sure service is the right word. maybe module?

Reply:
Done